bpf/kmesh/workload/include/authz.h

```diff
@@ -194,7 +317,7 @@ do_auth(Istio__Security__Authorization *policy, struct xdp_info *info, struct bp
         if (!rule) {
             continue;
         }
-        if (rule_match_check(rule, info, tuple_info) == MATCHED) {
+        if (rule_match_check(ctx, rule, policy->action) == MATCHED) {
```

Comment:
Match should not be returned. If the processing is successful in tail_call, will use taill_call result. If cant handle (the configuration is abnormal), bypass the socket directly.